lib/events/api.go

```diff
@@ -141,6 +141,10 @@ type IAuditLog interface {
 	// The only mandatory requirement is a date range (UTC). Results must always
 	// show up sorted by date (newest first)
 	SearchEvents(fromUTC, toUTC time.Time, query string) ([]EventFields, error)
```

Comment:
why not simply extend query to add query type for session type events instead of adding extra method?